go.mod

```diff
@@ -36,7 +36,7 @@ require (
 	github.com/stretchr/testify v1.8.1
 	github.com/tangzero/inflector v1.0.0
 	github.com/withfig/autocomplete-tools/packages/cobra v1.2.0
-	go.mongodb.org/atlas v0.20.0
+	go.mongodb.org/atlas v0.20.1-0.20230105135126-32300174f475
```

Comment:
We generally make a new go client release when we need it instead of pinging a specific not-released commit. I will release a 0.20.2 today so that you can use that release.

Reply:
Thank you

Reply:
I will release it this afternoon. Waiting for the go-ahead from terraform

Reply:
v0.21.0 has been released